core/trino-main/src/test/java/io/trino/sql/planner/TestRecursiveCte.java

```diff
@@ -61,15 +61,15 @@ public void testRecursiveQuery()
                                 // base term
                                 project(project(project(
                                         ImmutableMap.of("expr", expression("1")),
-                                        values()))),
+                                        project(values())))),
```

Comment:
What's in this new projection?

Reply:
This new projection is a result of `planWindowDefinitions()`. There are no window definitions in this query, so it is just an identity projection. It shows up in the plan, because it is in stage `CREATED` (not optimized).
As I said in previous comments, I'm going to remove `planWindowDefinitions()`, and this projection will disappear.